### PR TITLE
Fix page width of the API pages

### PR DESCRIPTION
**Pull request by dawehner**
Task: #

* [x] Ready for review
* [x] Ready for merge

### Notes

The API pages should have full width, as there is a LOT of content displayed

#### Before
screen shot 2017-07-15 at 21 50 36

#### After
screen shot 2017-07-15 at 21 50 50

### CODE REVIEW

File: themes/materialize_contenta/css/materialize-drupal.css
Comment (e0ipso) [inline themes/materialize_contenta/css/materialize-drupal.css:8799]: I don't think we need source maps for prod.

File: themes/materialize_contenta/materialize_contenta.theme
Comment (e0ipso) [inline themes/materialize_contenta/materialize_contenta.theme:15]: That's a cool snippet.

File: themes/materialize_contenta/css/materialize-drupal.css
Comment (e0ipso) [reply themes/materialize_contenta/css/materialize-drupal.css:8799]: I don't want to block the PR on this. I will fix in a subsequent PR.

**Review by e0ipso — APPROVED** [review]

File: themes/materialize_contenta/css/materialize-drupal.css
Comment (dawehner) [reply themes/materialize_contenta/css/materialize-drupal.css:8799]: Just edit the files right before the commit :)

Comment (e0ipso) [reply themes/materialize_contenta/css/materialize-drupal.css:8799]: @dawehner it's easier to do a separate commit when using the GH merge button. :-P

Comment (dawehner) [reply themes/materialize_contenta/css/materialize-drupal.css:8799]: Fair :)

File: themes/materialize_contenta/materialize_contenta.theme
Comment (dawehner) [reply themes/materialize_contenta/materialize_contenta.theme:15]: Well theming based upon routes is a little bit lame IMHO, but its still better than paths.

Comment (e0ipso) [reply themes/materialize_contenta/materialize_contenta.theme:15]: Isn't that what landing pages are?

Comment (dawehner) [reply themes/materialize_contenta/materialize_contenta.theme:15]: I guess, at least the route name doesn't suddenly change.

**Pull request merged**
